profile/main.go

```diff
@@ -412,21 +461,83 @@ func handler(request events.APIGatewayProxyRequest) (events.APIGatewayProxyRespo
 
 		logHealth(client, ctx, userId, isServiceRunning)
 		if !isServiceRunning {
-			setProfileStatusBlocked(client, ctx, userId, "BLOCKED")
+			profilesSkipped.ServiceDown = append(profilesSkipped.ServiceDown, userId)
+			logProfileSkipped(client, ctx, userId, "Service Down")
+			setProfileStatusBlocked(client, ctx, userId, Constants["STATUS_BLOCKED"])
```

Comment:
Here instead of constant we have to give reason behind block which is `Profile Service Down`